src/diffusers/utils/import_utils.py

```diff
@@ -123,6 +123,12 @@
 except importlib_metadata.PackageNotFoundError:
     _inflect_available = False
 
+_huggingface_hub_available = importlib.util.find_spec("huggingface_hub") is not None
```

Comment:
Correct, though `huggingface_hub` is a already a required dependency by default.